integration/consumers/tsc/src/main.ts

```diff
@@ -1,10 +1,22 @@
 import { NgModule } from '@angular/core';
 import { FooComponent, BarComponent } from 'sample-custom';
+import { CoreConsumerModule } from './core-consumer/core-consumer.module';
+import { CustomConsulerModule } from './custom-consumer/custom-consumer.module';
+//import { JsxConsumerModule } from './jsx-consumer/jsx-consumer.module';
+import { MaterialConsumerModule } from './material-consumer/material-consumer.module';
+import { ModuleImportDirective } from './module-imports';
 
 @NgModule({
+  imports: [
+    CoreConsumerModule,
+    CustomConsulerModule,
+    //JsxConsumerModule,
```

Comment:
@KyleMcNutt When commented-in, it produces a build error:

```bash
$ ngc -p integration/consumers/tsc/tsconfig.json
: 'react-integration-test' is not a known element:
1. If 'react-integration-test' is an Angular component, then verify that it is part of this module.
2. If 'react-integration-test' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas'of this component to suppress this message. ("
[ERROR ->]<react-integration-test></react-integration-test>
```

Can you maybe take a look?